src/start_menu.c

```diff
@@ -200,6 +204,7 @@ static const struct MenuAction sStartMenuItems[] =
     [MENU_ACTION_RETIRE_FRONTIER] = {gText_MenuRetire,  {.u8_void = StartMenuBattlePyramidRetireCallback}},
     [MENU_ACTION_PYRAMID_BAG]     = {gText_MenuBag,     {.u8_void = StartMenuBattlePyramidBagCallback}},
     [MENU_ACTION_DEBUG]           = {sText_MenuDebug,   {.u8_void = StartMenuDebugCallback}},
+    [MENU_ACTION_DEXNAV]          = {gText_MenuDexNav,  {.u8_void = StartMenuDexNavCallback}},
```

Comment:
I'm personally not sure if we should be adding DexNav support for the start menu considering it'll overflow by default. This is something we should discuss on Discord perhaps

Reply:
so that this doesn't block the PR, I think adding to the config 

> hey, turning this on will add this to the start menu

and letting the user deal with it is fine

Reply:
FLAG_SYS_DEXNAV_GET needs to be set for it to show up so it does not overflow by default. I will add a comment about that here. The code needs to exist as a reference for the entry point of the GUI, also